### ISSUE TITLE
[@angular-eslint/template/banana-in-box] Remove this rule. It is a duplicate of invalidBananaInBox

### ISSUE THREAD

**Issue by DerZyklop** [angular-eslint/angular-eslint#1080]
**Description and reproduction of the issue**

Correct me if I am wrong, but [@angular-eslint/template/banana-in-box](https://github.com/angular-eslint/angular-eslint/blob/37e4cb20269fa184c379823780ed43c2dba17db6/packages/eslint-plugin-template/src/rules/banana-in-a-box.ts) does the same as [invalidBananaInBox](https://angular.io/extended-diagnostics/NG8101).

@angular-eslint/template/banana-in-box was awesome and saved us a lot of hustle. But since Angular 13.2 i can see no benefit of its existence. I think it should be removed. 

.eslintrc.json
```JSON
{
  "rules": {
    "@angular-eslint/template/banana-in-box": ["error"]
  }
}
```

app.component.ts
```ts
@Component({
  selector: 'app',
  templateUrl: './app.component.html',
})
export class AppComponent {
  private something : string = 'Hello';
}
```

```html
<div ([ngModel])="something"></div>
```

⬆ Gives two Errors:
```
In the two-way binding syntax the parentheses should be inside the brackets, ex. '[(ngModel)]="true"'.
        Find more at https://angular.io/guide/two-way-bindingngtsc(-998101)
        Error occurs in the template of component AppComponent.
----
Invalid binding syntax. Use [(expr)] instead
eslint[@angular-eslint/template/banana-in-box](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/docs/rules/banana-in-box.md)
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/eslint-plugin`    | `14.0.0` |
| `@typescript-eslint/parser`        | `5.30.4` |
| `ESLint`                           | `8.18.0` |
| `node`                             | `16.13` |

```sh
# Please run `npx ng version` in your project and paste the full output here:


Angular CLI: 14.0.4
Node: 16.13.2
Package Manager: npm 6.14.12 
OS: darwin x64

Angular: 14.0.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, platform-server, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1400.4
@angular-devkit/build-angular   14.0.4
@angular-devkit/core            14.0.4
@angular-devkit/schematics      14.0.4
@angular/fire                   6.1.5
@schematics/angular             14.0.4
rxjs                            7.5.5
typescript                      4.7.4
webpack                         5.73.0

```

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Duplicate of https://github.com/angular-eslint/angular-eslint/issues/660

The compiler check does not yet have auto-fixing, we will deprecate the lint rule once it can match that very useful capability of the lint rule

**Comment by DerZyklop** [comment]
@JamesHenry thanks and sorry ツ ✌
